lms/urls.py

```diff
@@ -586,6 +587,11 @@
     urlpatterns += (
         url(r'', include('third_party_auth.urls')),
         url(r'^login_oauth_token/(?P<backend>[^/]+)/$', 'student.views.login_oauth_token'),
+        url(
+            r'^exchange_oauth_token/$',
+            oauth_exchange.views.AccessTokenExchangeView.as_view(),
+            name="exchange_oauth_token"
+        ),
```

Comment:
Why did you choose to put this as a top level endpoint in lms instead of within the third_party_auth library?  The 3rd party library code is already openedX specific.  It seems all of our auth endpoints should be namespaces within something like "auth".